lms/static/sass/discussion/utilities/_variables.scss

```diff
@@ -6,3 +10,8 @@ $forum-color-following: $blue;
 $forum-color-staff: $blue;
 $forum-color-community-ta: $green-d1;
 $forum-color-marked-answer: $green-d1;
+
+// post images
+$post-image-dimension: ($baseline*3);  // image size  + margin
```

Comment:
Good variables, but can we hint that these control the size more, perhaps something like:

```scss
$post-image-size: [value]; 
```

or

```scss
$post-image-width: [value];
$post-image-height: $post-image-width;  
```